### PR TITLE
fix #35135 ool.ru

### LINKED ISSUE #35135: ool.ru
### Issue URL (Incorrect Blocking)
[https://www.ool.ru/news/krym/obshchestvo/nemedlenno-i-bez-vsyakikh-usloviy-ukraintsy-prosyat-zelenskogo-vernut-vodu-v-krym-/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.ool.ru%2Fnews%2Fkrym%2Fobshchestvo%2Fnemedlenno-i-bez-vsyakikh-usloviy-ukraintsy-prosyat-zelenskogo-vernut-vodu-v-krym-%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.0.2693.6661
Browser: | Chrome
AdGuard driver: | TDI
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Self-destructing third-party cookies (2880), Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>EasyList China,<br>EasyList Italy,<br>Latvian List,<br>AdGuard Annoyances,<br>AdGuard Base,<br>Estonian List,<br>AdGuard Tracking Protection,<br>AdGuard Social Media,<br>AdGuard German,<br>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/35135

### CODE REVIEW

**Review by BlazDT — CHANGES_REQUESTED** [review]
Ads on main page:
||ool.ru/banners-img^
||ool.ru/upload/rk^
ool.ru##div[id^="swiffycontainer_work_"]

Fix for big empty boxes on main pages when dev tools are open and you zoom out, happens with non-modified site too, so would put it into Annoyances/tweaks:
ool.ru#$#.row > div[style*="height: 3632px;"] { height:calc(100%)!important; }

**Review by BlazDT — APPROVED** [review]

File: RussianFilter/sections/specific.txt [hunk 1]
```diff
@@ -1634,7 +1634,10 @@ finam.info##.banner-right-second
 bighd.net,kinoprofi.net,kinoprofi.org,vseriale.tv##.banner-sidebar
 нашбийск.рф##.banner-src
 eva.ru##.banner-style
-orsk.ru##.banner-top
+ool.ru,orsk.ru##.banner-top
+||ool.ru/banners-img^
+||ool.ru/upload/rk^
```
Comment (Alex-302) [inline RussianFilter/sections/specific.txt]: add new domain to `/upload/rk/*$domain=`
Better leave `/` instead of `^`

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/tweaks.txt [hunk 2]
```diff
@@ -9,6 +9,8 @@ linksht.com#$##originalLink { display: block!important; }
 reddit.com#$#body > .footer-parent { height: calc(100%)!important; padding:0px!important; }
 ! https://github.com/AdguardTeam/AdguardFilters/issues/22766
 classicfm.com#$#.radioplayer { height:calc(100%)!important; }
+! https://github.com/AdguardTeam/AdguardFilters/issues/35135
+ool.ru#$#.row > div[style*="height: 3632px;"] { height:calc(100%)!important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/tweaks.txt]: it is not required on my end. Please check again.

Comment (BlazDT) [reply AnnoyancesFilter/sections/tweaks.txt]: Checked again without rule, see screenshot:

ool ru

I see height is not always that high and also requires sub-element to be modified:
`ool.ru#$#.row div.sameblockheight[style^="height: "] { height:calc(100%)!important; }`

**Review by BlazDT — APPROVED** [review]

File: AnnoyancesFilter/sections/tweaks.txt (hunk 2 above)
Comment (Alex-302) [reply AnnoyancesFilter/sections/tweaks.txt]: @zloyden

Comment (zloyden) [reply AnnoyancesFilter/sections/tweaks.txt]: I had the same thing that's why I added two tweaks from BlazDT. With this rules the page looks ok.
Here is a screenshot without tweaks.
<details><summary>Screenshot:</summary>


</details><br/>

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**